### ISSUE TITLE
Cast Return and it put party in water

### ISSUE THREAD

**Issue by depauldoug** [gameboy9/DW3Randomizer#22]
Hello

Not sure if this is known or not, but I cast return to a town (Kanave) and it put my party into the water. The town is on a south border of land. I cannot move east, west, or south. I can only move north which is the town.  I do not have the boat.   I attached a word document for your review. 

File name was: DW3Random_2063678918_3Zy@R.nes

[Return_into_water.docx](https://github.com/gameboy9/DW3Randomizer/files/3163714/Return_into_water.docx)

**Comment by gameboy9** [comment]
Yeah I need to do a better job with the Return business... I'll look into it.
